driver-sync/src/test/functional/com/mongodb/client/ChangeStreamProseTest.java

```diff
@@ -48,6 +49,7 @@
 // See https://github.com/mongodb/specifications/tree/master/source/change-streams/tests/README.rst#prose-tests
 public class ChangeStreamProseTest extends DatabaseTestCase {
     private BsonDocument failPointDocument;
+    private volatile boolean interruptedExceptionOccurred = false;
```

Comment:
Since it only applies to one test, move this to the ChangeStreamWatchThread class

Reply:
Done